### ISSUE TITLE
ACE Towing In MP

### ISSUE THREAD

**Issue by OHoltan** [acemod/ACE3#9619]
**Mods (complete and add to the following information):**
- **Arma 3:** 2.14.150957 Stable
- **CBA:** 3.16.1.231025 Stable
- **ACE3:** 3.16.1.74 Stable

Mods:

**Description:**
In Multiplayer ACE towing does not seem to work, was tested in Single-player and worked.

**Steps to reproduce:**

1.  Join MP
2. Attach 2 Vehicles with ropes (we used 2x Offroads)
3. Try to tow 

**Expected behavior:**
Expect the vehicle to be towed

**Where did the issue occur?**
- Dedicated 

**Log Files:**
Not able to locate (using FASTER on win server datacenter 2w22)

**Comment by BaerMitUmlaut** [comment]
A script error popped up when I tried reproducing this issue, I assume this is related or the cause:
```
 0:22:15 Error in expression < ["ace_towing_hook", objNull];

_parent removeEventHandler ["Deleted", _hook get>
 0:22:15   Error position: <removeEventHandler ["Deleted", _hook get>
 0:22:15   Error Type Any, expected Number
 0:22:15 File z\ace\addons\towing\functions\fnc_detach.sqf..., line 23
 0:22:15  ➥ Context: 	[] L127 (z\ace\addons\towing\functions\fnc_towStateMachinePFH.sqf)
	[] L44 (z\ace\addons\towing\functions\fnc_detach.sqf)
	[] L292 (/x/cba/addons/events/fnc_targetEvent.sqf)
 ```

**Comment by jonpas** [comment]
There is a PR open fixing more than a few towing issues.

**Comment by Dystopian** [comment]
See #9007.

**Comment by LinkIsGrim** [comment]
Solved in #9007
